tests/integration/component.go

```diff
@@ -902,5 +902,56 @@ func componentTests(args ...string) {
 			oc.VerifyResourceDeleted("pvc", "storage-2", project)
 			oc.VerifyResourceDeleted("dc", cmpName, project)
 		})
+
 	})
+
+	/*
+		Commenting the tests till this PR gets merged https://github.com/openshift/odo-init-image/pull/70
```

Comment:
The PR is merged. Holding till a new image is released.
/hold

Reply:
There are some issues with releasing new image. If this PR doesn't affect normal functionality and other tests we can merge. Otherwise, it might stay here for a while, and it will need a lot of work rebasing this

Reply:
OK then let's merge.